app/api/images/owner.images.controller.js

```diff
@@ -79,32 +83,43 @@ exports.getAttributes = utils.route(async (req, res) => {
+                    THEN json_build_object('image_url', NULL,
+                                        'tiles', json_build_object('type', 'iiif', 'url', CONCAT(iiif_data->>'image_service3_url', '/info.json')),
+                                        'model_3d_url', CONCAT('${config.apiUrl}/data/collections/', collection_id,'/gltf/',images.id,'.gltf'))
+                    WHEN iiif_data->>'regionByPx' IS NOT NULL AND geolocalisation.region_px IS NOT NULL
+                      THEN json_build_object('image_url', CONCAT((iiif_data->>'image_service3_url'), '/', regexp_replace(iiif_data->>'regionByPx','[\\[\\]]', '', 'g'),'/200,/0/default.jpg'),
+                                                        'tiles', json_build_object('type', 'iiif', 'url', CONCAT(iiif_data->>'image_service3_url', '/info.json')),
+                                                        'model_3d_url', CONCAT('${config.apiUrl}/data/collections/', collection_id,'/gltf/',images.id, '_', geolocalisation.region_px[1], '_', geolocalisation.region_px[2], '_', geolocalisation.region_px[3], '_', geolocalisation.region_px[4], '.gltf'),
+                                                        'regionByPx', iiif_data->'regionByPx')
+                    WHEN iiif_data->>'regionByPx' IS NOT NULL AND geolocalisation.region_px IS NULL
+                      THEN json_build_object('image_url', CONCAT((iiif_data->>'image_service3_url'), '/', regexp_replace(iiif_data->>'regionByPx','[\\[\\]]', '', 'g'),'/200,/0/default.jpg'),
+                                        'tiles', json_build_object('type', 'iiif', 'url', CONCAT(iiif_data->>'image_service3_url', '/info.json')),
+                                        'model_3d_url', CONCAT('${config.apiUrl}/data/collections/', collection_id,'/gltf/',images.id,'.gltf'),
+                                        'regionByPx', iiif_data->'regionByPx')
+                    WHEN iiif_data->>'regionByPx' IS NULL AND geolocalisation.region_px IS NOT NULL
+                      THEN json_build_object('image_url', CONCAT((iiif_data->>'image_service3_url'), '/full/200,/0/default.jpg'),
+                                        'tiles', json_build_object('type', 'iiif', 'url', CONCAT(iiif_data->>'image_service3_url', '/info.json')),
+                                        'model_3d_url', CONCAT('${config.apiUrl}/data/collections/', collection_id,'/gltf/',images.id, '_', geolocalisation.region_px[1], '_', geolocalisation.region_px[2], '_', geolocalisation.region_px[3], '_', geolocalisation.region_px[4], '.gltf'))
+                    ELSE json_build_object('image_url', CONCAT((iiif_data->>'image_service3_url'), '/full/200,/0/default.jpg'),
+                                       'tiles', json_build_object('type', 'iiif', 'url', CONCAT(iiif_data->>'image_service3_url', '/info.json')),
+                                       'model_3d_url', CONCAT('${config.apiUrl}/data/collections/', collection_id,'/gltf/',images.id,'.gltf'))
+                    END
```

Comment:
we have to refactor those json_build_object commands, it is now really hard to read. 
We can maybe check together once how to refactor it